plugins/Process.cpp

```diff
@@ -9,6 +9,8 @@
 
 #include <chrono>
 
+constexpr uint16_t findPatternBufferSize = 32768;
```

Comment:
This still doesn't explain, why it has to be this specific vlaue...

And if this variable is only ever used inside `findPattern`, then I would make this a local variable to that function.

Reply:
It doesn't have to be a specific value.

In fact, I could make the variable a parameter to the function.

Reply:
Well if it doesn't have to be a specific value, how did you arrive at this particular number? It looks oddly specific to me xD

I don't think that making this a function parameter is a good idea. This seems like an implementation detail that the caller shouldn't have to worry about :thinking:

Reply:
I chose a number that appears a good balance for the buffer size.

Reply:
I guess if it was `33000` instead folks like me wouldn't start wondering how that number came to existence ^^

Reply:
I wrote a comment explaining that it's 32 KiB.